spec/uploader/callback_spec.rb

```diff
@@ -3,19 +3,27 @@
 describe CarrierWave::Uploader do
 
   it "should keep callbacks on different classes isolated" do
+    default_before_callbacks = [
+      :check_whitelist!,
```

Comment:
Might be worth changing `check_whitelist!` to `check_extension_whitelist!` and similar. Happy to do it as a separate PR though.